pkg/kubelet/status/status_manager.go

```diff
@@ -482,6 +487,13 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 		podName:      pod.Name,
 		podNamespace: pod.Namespace,
 	}
+
+	if cachedStatus.at.IsZero() {
+		newStatus.at = time.Now()
+	} else {
+		newStatus.at = cachedStatus.at
```

Comment:
This essentially ensures we measure the longest outstanding status update for a pod, right?  E.g. if multiple status updates end up batched, we want to measure how long it took from the first update.

Can you add a brief comment?

Reply:
Will do